client/src/components/JobCard.vue

```diff
@@ -116,6 +120,21 @@
         return classes;
       },
 
+      jobSectorName() {
+        return this.job.project.customer.sector.name;
+      },
```

Comment:
To know if a job is in France or somewhere else, I've taken "project.customer.sector.name", as shown in octopod and in the jobs list payload from https://jobs.octo.com/.